### PR TITLE
add gitbook-like optional stylesheet

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review

#### What are the relevant tickets?

none

#### What's this PR do?

adds a gitbook-inspired selectable stylesheet to the project, to give a basic idea of what this could look like.

#### How should this be manually tested?

should be able to switch to it, should look like the screenshot below.

#### Screenshots (if appropriate)

@pdpinch @Ferdi I implemented so far basically restyling the sidenav to look like gitbook, background colors, and a few other things. Is this what you were thinking, or should I add other parts too?

Screenshot from 2020-03-06 14-20-35

### CODE REVIEW

File: src/css/gitbook.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  .desktop-nav {
+    border-right: 1px solid $gitbook-border-light;
+
+    .course-nav-list-item.active {
+      background-color: white;
+      border: 1px solid $gitbook-border-light;
+      border-right: none;
+
+      a {
```
Comment (gumaerc) [inline src/css/gitbook.scss]: This rule causes all links under a list item to be colored blue, regardless of hierarchy tier:

image

I think maybe the text should only be blue on the first (selected) link.  I was thinking you could add an ID only to child links like this in `nav_item.html`:
```
<a class="text-uppercase text-dark{{ if $hasParent }} course-nav-link-child{{ end }}{{ if $isMenuCurrent }} active{{ end }}"
```

Then in `gitbook.scss`, you could change the nested a selector to look like this:

```
      a:not(.course-nav-link-child) {
        color: $gitbook-blue !important;
      }
```

Comment (gumaerc) [reply src/css/gitbook.scss]: Also, just noticing that it's kind of a little odd how the child items wrap text there.

Comment (aliceriot) [reply src/css/gitbook.scss]: should have made it more clear! ferdi shared some feedback that I'm sitll working on, so things aren't quite final yet

Comment (gumaerc) [reply src/css/gitbook.scss]: Gotcha, I wasn't sure if you wanted a review or not yet but figured it couldn't hurt to give it a look.

**Comment by aliceriot** [comment]
fixed issues with the sidenav, and made the font styling the same for everything:

Screenshot from 2020-03-11 12-03-55

**Comment by gumaerc** [comment]
@aliceriot It doesn't look like any commits were pushed before you made your last comment, and I'm still seeing the same issues pulling it down locally.  Did you forget to push maybe?

**Comment by aliceriot** [comment]
I sure did :speak_no_evil:

**Pull request merged**